Trappist/src/Promact.Trappist.Core/Controllers/HomeController.cs

```diff
@@ -1,11 +1,17 @@
 ﻿using Microsoft.AspNetCore.Mvc;
-using Microsoft.AspNetCore.Authorization;
+using Microsoft.Extensions.Logging;
 
 namespace Promact.Trappist.Web.Controllers
 {
-    // [Authorize]
     public class HomeController : Controller
     {
+        readonly ILogger<HomeController> _logger;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the '_logger' field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1450)